staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -77,8 +77,6 @@ type Event struct {
 	// AuditLevel at which event was generated
 	Level Level
 
-	// Time the request reached the apiserver.
-	Timestamp metav1.Time
```

Comment:
>  I'm not sure whether we want to use CreationTimestamp for one (it could be argued for either...), 

I prefer use CreationTimestamp as "request timestamp", this time is also when the audit event is initialized.

> I think that either way what Timestamp means is not very clear, it should be something like RequestStartTimestamp or StartTimestamp

What about rename Timestamp to StageTimestamp?
And reset this filed for each stage?